Readme.md

```diff
@@ -225,134 +244,204 @@ is of interest to you. This will likely be done by your favorite IDE, probably
 just in time when you explore documentation. Nonetheless, we've
 implemented a couple of external renderers to help debug issues.
 
-WARNING:
-
-Many rendering methods current require papyri's own docs to be built and ingested
-first.
+> [!WARNING]
+> Many rendering methods currently require papyri's own docs to be built and ingested first.
 
 ```
 $ papyri gen examples/papyri.toml
 $ papyri ingest ~/.papyri/data/papyri_0.0.7  # or any current version
 ```
 
-Or you can try to pre-install an old papyri doc bundle
+Or you can try to pre-install an old papyri doc bundle:
 
 ```
 $ papyri install papyri
 ```
 
 ### Standalone HTML rendering
 
+To see the rendered documentation for all packages previously ingested, run
 
 ```bash
-$ papyri render  # render all the html pages statically in ~/.papyri/html
-$ papyri serve-static # start a http.server with the propoer root to serve above files.
+$ papyri serve
 ```
 
+This will start a live server that will render the pages on the fly.
+
+If you need to render static versions of the pages, use either of the following
+commands:
+
 ```bash
-$ papyri serve  # start a server that will render the pages on the fly (nice to debug or iterate on theme, rendering)
+$ papyri render  # render all the html pages statically in ~/.papyri/html
+$ papyri serve-static # start a http.server with the proper root to serve above files.
 ```
 
-### Ascii terminal rendering (experimental)
+### Rich terminal rendering
 
+To render the documentation for a single object on a terminal, use
 
 ```
-$ papyri ascii <fully qualified names> # try to render in the terminal.
+$ papyri rich <fully qualified name>
 ```
 
-For example,
+For example:
 
 ```
-$ papyri ascii numpy.linspace
+$ papyri rich numpy:einsum  # note the colon for the fully qualified name.
 ```
 
-The next step uses urwid to provide a browsable interface in terminal.
+To use the experimental interactive Textual interface in the terminal, use
 
 ```
-$ papyri browse <fully qualified name> # urwid documentation browser.
+$ papyri textual <fully qualified name>
+```
+
+### IPython extension
+
+To run `papyri` as an IPython extension, run:
+
 ```
+$ ipython --ext papyri.ipython
+```
+
+This will start an IPython session with an augmented `?` operator.
+
+### Jupyter extension
+
+In progress.
```

Comment:
I couldn't immediately find how to run papyri as a jupyter extension

Reply:
There is a sub-readme, and yes it's complicated, I would need to publish it as its own subpackage.